apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -641,6 +653,21 @@ static void tc_fs_vfs_opendir(void)
 	dirp = opendir(VFS_FOLDER_PATH);
 	TC_ASSERT_NEQ("opendir", dirp, NULL);
 	closedir(dirp);
+	/*Path doesnot exist */
+	dirp = opendir(INVALID_PATH);
+	TC_ASSERT_EQ("opendir", dirp, NULL);
+
+	dirp = opendir(DEV_PATH);
+	TC_ASSERT_NEQ("opendir", dirp, NULL);
+
+	/* Pseudo file system node covers error condition path is not a directory */
+	dirp = opendir(DEV_INVALID_DIR);
+	TC_ASSERT_EQ("opendir", dirp, NULL);
```

Comment:
1. resource loss with dirp, as it is used for new instance.
2. Does it not required to release  with closedir()?

Reply:
No, actually directory has not been opened (invalid path) covers the error condition. So we cannot close .

Reply:
The previous call, dirp = opendir(DEV_PATH);

Reply:
Okay will close that dirp.